src/Rules/Documentation.hs

```diff
@@ -33,10 +36,10 @@ documentationRules = do
         need $ map (root -/-) $ [html] ++ archives ++ pdfs
         need [ root -/- htmlRoot -/- "libraries" -/- "gen_contents_index" ]
         need [ root -/- htmlRoot -/- "libraries" -/- "prologue.txt" ]
-        need [manPagePath]
+        need [ root -/- manPagePath ]
 
 manPagePath :: FilePath
-manPagePath = "_build/docs/users_guide/build-man/ghc.1"
+manPagePath = "docs/users_guide/build-man/ghc.1"
```

Comment:
Why build docs outside the build directory?

Reply:
Ah, it looks like this is not a `Path`, it's a `Dir`!

Reply:
Yes, it's a relative path, the man page ends up being produced at `<build root>/docs/users_guide/build-man/ghc.1`. This however _is_ a file, not a directory.

Reply:
Aha, I see. Perhaps, we can call it `manPageBuildPath` or something like that? I just want to keep convention that `fooDir` correspond to relative directories and `fooPath` correspond to paths from the GHC tree. In an ideal future, we will introduce a type-level distinction between all these notions to avoid confusion.

Reply:
OK, let's call it `manPageBuildPath`, the name is crystal clear. I'll make this change in an upcoming commit.